core/src/surplus.ts

```diff
@@ -59,8 +59,10 @@ export const fetchSurplusAuctionByIndex = async function (
     auctionIndex: number
 ): Promise<SurplusAuction> {
     const contract = await getContract(network, 'MCD_FLAP');
+    const auctionDuration = await contract.tau();
     const auctionData = await contract.bids(auctionIndex);
-    const isAuctionCollected = new BigNumber(auctionData.end).eq(0);
+    const createdAt = await auctionData.end;
+    const isAuctionCollected = new BigNumber(createdAt).eq(0);
```

Comment:
`auctionData.end` is not `createdAt`. Moreover, there is already a property called `auctionEndDate` that contains date that you need to calculate `auctionStartDate`

Reply:
done